webnn/resources/utils.js

```diff
@@ -67,21 +67,21 @@ const loadTests = (operationName) => {
 };
 
 /**
- * Get exptected data and data type from given resources with output name.
+ * Get exptected suboutput resources from given resources with output name.
  * @param {Array} resources - An array of expected resources
  * @param {String} outputName - An output name
- * @returns {Array.<[Number[], String]>} An array of expected data array and data type
+ * @returns {Object} An object of expected suboutput resources
  */
-const getExpectedDataAndType = (resources, outputName) => {
+const getExpectedSubOutputResources = (resources, outputName) => {
```

Comment:
Propose just calling `getNamedResource`, since it gets a resource of the given name (and it doesn't appear to be "output" specific either).